templates/activity-stream/pocket-button/pocket-button.1.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "pocket-button",
```

Comment:
Can you add a top-level "$comment" field to the existing `on-save-recs` schema to indicate that it's deprecated in favor of `pocket-button`? Do we expect we'll continue to receive `on-save-recs` submissions from older Fx installations for a long time? Is there an expected date on which we could completely remove `on-save-recs` considering we only store 180 days of history?

Reply:
Will do on the comment. The legacy data won't be essential once there is a critical mass of users on Firefox releases sending the new ping. I think we're targeting to add this ping into FX84, so probably 2 release cycles after that, we can get rid of it. That would put us at FX86 on 2/23/21.